src/session/sidebar/row.rs

```diff
@@ -180,9 +183,93 @@ impl Row {
                     "session",
                 );
 
-                // Title label bindings
-                let title_binding = title_expression.bind(&*self_.title_label, "label", Some(chat));
-                bindings.push(title_binding);
+                let is_own_chat_expression = match chat.type_().user().map(User::id) {
+                    Some(interlocutor_id) => {
+                        let me_expression = gtk::PropertyExpression::new(
```

Comment:
I guess we really should handle the 'me' User a bit better, to avoid having complicating things unnecessarily. Is it ok for you fetching the user in the Sessions's constructor? Otherwise we can do it in another PR.

Reply:
Ok, I opened a separate PR. I think this is wiser.

Reply:
Ok, #208 is closed, so we handle `me` still as optional because we cannot know when it arrives.